### PR TITLE
[FIX] point_of_sale: change pos receipt item direction in rtl languages

### PR DESCRIPTION
**Pull request by AH-Yussef**
### Steps to reproduce:

1. In the general settings, add the Arabic(العربية) language
2. Install the POS (point of sales) module
3. Go to the administrator's profile and change Preferences > Languages to Arabic(العربية) 
4. Open a new shop/pos session
5. Add any product
6. Click on payment(دفعة)  button
7. Choose the cash option under the payment method (طريقة الدفع) 
8. Select a client (العميل) 
9. Click on validate (تصديق) 
10. Notice how the items format is incorrect, the quantity and price flipped and unreadable.

### Investigation:
- when switching to **Arabic** (an right to left language), the direction of the order line (price and quantity) is `ltr` so the price and the quantity were flipped

opw-3580386

### CODE REVIEW

**Comment by robinengels** [comment]
LGTM. 
@odoo/pos

**Review by robinengels — APPROVED** [review]
LGTM. 
@odoo/pos

File: addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/OrderReceipt.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
             <!-- Orderlines -->
 
-            <div class="orderlines">
+            <div class="orderlines" dir="ltr">
```
Comment (nle-odoo) [inline addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/OrderReceipt.xml:49]: small note: if this is used to force ltr direction, if it's possible, usually we would rather have the equivalent CSS (with directive telling RTLCSS to not inverse it):

```css
    direction: ltr/*rtl:ignore*/; /* rtlcss forced to keep ltr */
```

**Review by nle-odoo — APPROVED** [review]

**Comment by adgu-odoo** [comment]
@robodoo r+

**Pull request closed without merging**